examples/digital_fingerprinting/production/Dockerfile

```diff
@@ -13,35 +13,83 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG MORPHEUS_CONTAINER=nvcr.io/nvidia/morpheus/morpheus
-ARG MORPHEUS_CONTAINER_VERSION=v24.10.00-runtime
+ARG BASE_IMG=nvcr.io/nvidia/cuda
+ARG BASE_IMG_TAG=12.5.1-base-ubuntu22.04
 
-FROM ${MORPHEUS_CONTAINER}:${MORPHEUS_CONTAINER_VERSION} as base
+FROM ${BASE_IMG}:${BASE_IMG_TAG} as base
 
-# # Fix the entrypoint to work with different WORKDIR
-ENTRYPOINT [ "/opt/conda/bin/tini", "--", "/workspace/docker/entrypoint.sh" ]
+# Install necessary dependencies using apt-get
+RUN apt-get update && apt-get install -y \
+      git \
+      git-lfs \
+      wget \
+    && apt-get clean
 
+# Install miniconda
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O /tmp/miniconda.sh \
+    && bash /tmp/miniconda.sh -b -p /opt/conda \
+    && rm /tmp/miniconda.sh
+
+# Add conda to the PATH
+ENV PATH=/opt/conda/bin:$PATH
+
+# Install Mamba, a faster alternative to conda, within the base environment
+RUN conda install -y mamba -n base -c conda-forge
+
+# Create base environment
+RUN conda create -y --name morpheus-dfp python=3.10
+
+# Activate the environment (make it default for subsequent commands)
+RUN echo "source activate morpheus-dfp" >> ~/.bashrc
+
+# Set default shell to bash
 SHELL ["/bin/bash", "-c"]
 
-# Set the workdir to the DFP base folder
-WORKDIR /workspace/examples/digital_fingerprinting/
+# Default MORPHEUS_DFP_VERSION
+ARG MORPHEUS_DFP_VERSION=24.10.00a
+
+# Add conda channels required for the Morpheus dependencies
+RUN source activate morpheus-dfp \
+    && conda config --env --add channels conda-forge \
+    && conda config --env --add channels nvidia \
+    && conda config --env --add channels rapidsai \
+    && conda config --env --add channels pytorch
+
+# Install morpheus-dfp conda package
+RUN echo "MORPHEUS_DFP_VERSION is set to: ${MORPHEUS_DFP_VERSION}"
+ENV MORPHEUS_DFP_VERSION=${MORPHEUS_DFP_VERSION}
+RUN source activate morpheus-dfp \
+    && mamba install -y \
```

Comment:
We've been running into issues with `mamba` lately, and have settled on using `conda --solver=libmamba` as a replacement.

Reply:
Ack